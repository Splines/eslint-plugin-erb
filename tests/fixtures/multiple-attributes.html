<button type="button"
              class="btn btn-sm btn-secondary ms-2"
              id="lecture-basics-cancel"
              data-test="<%= course.other_id %>"
              onclick="abc"
              data-id="<%= lecture.id %>">
        <%= t("buttons.discard") %>
</button>


