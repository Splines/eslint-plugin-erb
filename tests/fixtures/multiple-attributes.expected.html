<button
  id="lecture-basics-cancel"
  type="button"
  class="btn btn-sm btn-secondary ms-2"
  data-id="<%= lecture.id %>"
  data-test="<%= course.other_id %>"
  onclick="abc"
>
  <%= t("buttons.discard") %>
</button>
